notification/notification.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 const (
-	alertPushEndpoint = "/api/v1/alerts"
+	AlertPushEndpoint = "/api/v1/alerts"
```

Comment:
This constant doesn't really have to be in that package any longer.
Let's just move it to an unexported constant in `cmd/prometheus/config.go`.